Cards/MadGraph5_aMCatNLO/W/WtoLNu-2Jets_0J_amcatnloFXFX-pythia8/WtoLNu-2Jets_0J_amcatnloFXFX-pythia8_madspin_card.dat

```diff
@@ -0,0 +1,9 @@
+set ms_dir ./madspingrid
+
+set max_running_process 1
+set onlyhelicity True
+
+decay w+ > ta+ vt
+decay w- > ta- vt~
```

Comment:
I think these "decay XXX" lines were no longer needed when we do "set onlyhelicity True"

Reply:
Removed. Thanks for pointing out!